templates/admin/user/edit.tmpl

```diff
@@ -95,7 +95,7 @@
 						<input name="allow_git_hook" type="checkbox" {{if .User.CanEditGitHook}}checked{{end}} {{if DisableGitHooks}}disabled{{end}}>
 					</div>
 				</div>
-				<div class="inline field">
+				<div class="inline field" {{if or (DisableImportLocal) (.DisableMigrations)}}hidden{{end}}>
```

Comment:
not `and` ?

Reply:
If migrations are disabled, DisableImportLocal seems not be used (its migration parameter); thats why or should be ok here.

Reply:
I would remove DisableImportLocal check since: DisableImportLocal=true&&DisableMigrations=false should still show migration option!

Reply:
I would not remove it because DisableImportLocal=true||DisableMigrations=false hides it as it should :)

Reply:
@pboguslawski i think you did not understand the meaning of ImportLocal ... it means you are not allowed to import "local paths" to git repos.

Reply:
> it means you are not allowed to import "local paths" to git repos

The code says something different:

https://github.com/go-gitea/gitea/blob/master/custom/conf/app.example.ini#L535
https://github.com/go-gitea/gitea/blob/master/modules/setting/setting.go#L771
https://github.com/go-gitea/gitea/blob/master/modules/templates/helper.go#L229

Condition...

`{{if or (DisableImportLocal) (.DisableMigrations)}}hidden{{end}}`

...is fine and was tested by us during devel.